Makefile

```diff
@@ -36,9 +36,9 @@ test-unit:
 test-e2e:
 	./test/e2e-tests.sh
 
-# Run E2E tests from the current repo for serving+eventing+knativeKafka
+# Run the Kafka operator e2e hook
 test-e2e-with-kafka:
-	INSTALL_KAFKA=true TEST_KNATIVE_KAFKA=true ./test/e2e-tests.sh
+	TEST_KNATIVE_KAFKA=true ./test/e2e-tests.sh
```

Comment:
Don't we need to install kafka here too?

Reply:
```
if [[ $TEST_KNATIVE_KAFKA == true ]]; then
  (( !failed )) && install_strimzi || failed=6
  (( !failed )) && serverless_operator_kafka_e2e_tests || failed=7
fi
```
no, the target depends on `TEST_KNATIVE_KAFKA` and will install strimzi already at that point

Reply:
See comment below, I think you actually need the INSTALL flag too to make sure Kafka gets installed for the downstream tests.